ts/Series/Lollipop/LollipopSeries.ts

```diff
@@ -0,0 +1,253 @@
+/* *
+ *
+ *  (c) 2010-2020 Sebastian Bochan, Rafal Sebestjanski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type LollipopSeriesOptions from './LollipopSeriesOptions';
+import LollipopPoint from './LollipopPoint.js';
+import AreaSeries from '../Area/AreaSeries.js';
+const { prototype: areaProto } = AreaSeries;
```

Comment:
Please use the deconstruction on BaseSeries for all types: Area, Dumbbell and Column.